app/models/pawn.rb

```diff
@@ -1,11 +1,42 @@
 class Pawn < Piece
 
+  def valid_move?(x, y)
+    unless super(x, y)
+      return false
+    end
+    x_distance = x_position - x
+    y_distance = y_position - y
+    if occupied?(x, y)
+      if color == 'white'
+        x_distance.abs == 1 && y_distance == 1
+      else
+        (-x_distance).abs == 1 && -y_distance == 1
```

Comment:
don't think you need to inverse the x_distance cause `abs` will do that for you.